src/providers/github/Github.ts

```diff
@@ -149,64 +192,34 @@ export class Github {
 
   /**
    * Create a Github release
-   * - With `assetsDir`, all its files will be uploaded as release assets
    * @param tag "v0.2.0"
    * @param options
    */
-  async createReleaseAndUploadAssets(
+  async createRelease(
     tag: string,
     options?: {
       body?: string;
       prerelease?: boolean;
-      assetsDir?: string;
-      ignorePattern?: RegExp;
     }
   ): Promise<void> {
-    const { body, prerelease, assetsDir, ignorePattern } = options || {};
-    const release = await this.octokit.repos
-      .createRelease({
+    const { body, prerelease } = options || {};
+    await this.octokit
+      .request("POST /repos/{owner}/{repo}/releases", {
         owner: this.owner,
         repo: this.repo,
         tag_name: tag,
         name: tag,
         body,
-        prerelease
+        prerelease,
+        generate_release_notes: true,
+        headers: {
+          "X-GitHub-Api-Version": "2022-11-28"
+        }
       })
       .catch(e => {
         e.message = `Error creating release: ${e.message}`;
         throw e;
       });
-
-    if (assetsDir)
```

Comment:
Could you clarify why this code removal?

Reply:
We actually do not use the generated assets for anything. It slows down the release creation and we also need to remove it for the multi-variants as it is not feasible to have 20+ assets per release